wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandRobot.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+ * The CommandRobot class extends the {@link TimedRobot} class in order to ease functionality for
+ * Command Based Programming.
+ */
+public class CommandRobot extends TimedRobot {
+  private static final double kSchedulerOffset = 0.005;
+
+  /**
+   * Constructor for CommandRobot.
+   *
+   * @param mainPeriod periodic rate of the Main Robot Loop in ms.
+   * @param schedulerPeriod periodic rate of the CommandScheduler in ms.
```

Comment:
Aren't these in seconds?

Reply:
yes, I mixed up the java and cpp documentation lol